api/serverless.yml

```diff
@@ -415,24 +415,24 @@ functions:
         handler: src/components/bookmark/routes.create
         events:
             - http:
-                  path: ${self:custom.versions.v1}/publications/{id}/bookmark
+                  path: ${self:custom.versions.v1}/bookmarks
                   method: POST
                   cors: true
-    removeBookmark:
-        handler: src/components/bookmark/routes.remove
+    deleteBookmark:
+        handler: src/components/bookmark/routes.deleteBookmark
         events:
             - http:
-                  path: ${self:custom.versions.v1}/publications/{id}/bookmark
+                  path: ${self:custom.versions.v1}/bookmarks/{id}
                   method: DELETE
                   cors: true
     getBookmark:
         handler: src/components/bookmark/routes.get
         events:
             - http:
-                  path: ${self:custom.versions.v1}/publications/{id}/bookmark
+                  path: ${self:custom.versions.v1}/bookmarks/{type}/{entityId}
```

Comment:
this endpoint should look the same as the DELETE endpoint: /bookmarks/{id}

this endpoint requires the user to be logged in in order to be able to delete a bookmark, that means we have all the info we need: the bookmark id and the user id.

Reply:
I removed this endpoint since the UI won't be using it. As you recommended below, it now hits getMany (previously getAll) with filters to check if an entity is bookmarked